airflow/api_fastapi/common/parameters.py

```diff
@@ -112,6 +112,34 @@ def depends(self, only_active: bool = True) -> _OnlyActiveFilter:
         return self.set_value(only_active)
 
 
+class _NoneFilter(BaseParam[bool]):
+    """check if a column is none or not."""
+
+    def __init__(self, attribute: ColumnElement, value: bool | None = None, skip_none: bool = True) -> None:
+        super().__init__(value, skip_none)
+        self.attribute: ColumnElement = attribute
+        self.value: bool | None = value
+
+    def to_orm(self, select: Select) -> Select:
+        if not self.value and self.skip_none:
+            return select
+        return select.where(self.attribute.is_(None))
+
+    def depends(self, *args: Any, **kwargs: Any) -> Self:
+        raise NotImplementedError("Use search_param_factory instead , depends is not implemented.")
+
+
+def none_param_factory(
+    attribute: ColumnElement,
+    none_filter_name: str,
+    skip_none: bool = True,
+) -> Callable[[bool | None], _NoneFilter]:
+    def depends_none(value: bool | None = Query(alias=none_filter_name, default=None)) -> _NoneFilter:
```

Comment:
```suggestion
    def depends_none(value: bool = Query(alias=none_filter_name, default=False)) -> _NoneFilter:
```

`only_active=False` return `all`, same as `only_active=None`. We don't need both `None` and `False` I think

Reply:
Maybe a more general approach is to `active=True` `active=False` `active=None`. 

- None is ignored and no filtering happen, returning all resources
- True, only take `active` ones, i.e end date is None
- False only take `inactive` ones, i.e end data is not None

This will allow retrieving only `inactive` backfills (which is not possible at the moment)

Reply:
Added the 3 cases.